CMakeLists.txt

```diff
@@ -44,10 +38,7 @@ set(VALID_BUILD_TYPES
 set(CMAKE_BUILD_TYPE
     "${DEFBUILD}"
     CACHE STRING "Choose the type of build, options are: ${VALID_BUILD_TYPES}")
-set_property(
-  CACHE CMAKE_BUILD_TYPE
-  PROPERTY STRINGS ${VALID_BUILD_TYPES}
- )
+set_property(CACHE CMAKE_BUILD_TYPE PROPERTY STRINGS ${VALID_BUILD_TYPES})
```

Comment:
I won't block the merge on this, but I noticed there were style changes throughout this `CMakeLists.txt` that changed the formatting of commands from being multi-line to being one line, like here. We should revert to the multi-line convention for consistency.

Reply:
Either @mathiaswagner or @bjoo can look at some automatic clang-format for `CMakeLists.txt` at some point. No point in obsessing over formatting unrelated to actual code.